### DISCUSSION TITLE
Multi user?

### DISCUSSION THREAD

**Discussion by fredamn76** [music-assistant/support#800] [Q&A] [answered]
Is multi user support on the roadmap? My family members have their own Spotify accounts and it would be great to change user

**Comment by marcelveldt** [comment, marked as answer]
Multi user NO 
Multi account YES

**Reply by fredamn76** [reply]
Multi account sounds great. Any timeline?

**Reply by marcelveldt** [reply]
Well the multi account support is already in the backend for some time now but the frontend (configuration) is missing. The HA config flow wizard is too limited to properly implement it so I'm probably just going to move the configuration of music providers to the dedicated web interface.

I hope to finish this somewhere in the next 2 weeks.

**Reply by fredamn76** [reply]
Sounds great! Thank you for your hard work.

**Reply by wjbeckett** [reply]
Oh that’s amazing. 
i assume this will cover multiple YouTube Music account’s too, giving user’s the ability to select which account they play music from (and also have all the playlists (including super mix) etc too)?
